notebooks/baseball.ipynb

```diff
@@ -1,5 +1,16 @@
 {
```

Comment:
<p>Just a quick question: is it the right place in the notebook to explain why the <code>[-1::-1]</code> exists? Apart from this one little optional nitpick, I don't have other comments on the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc4/pull/203/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>